src/serac/physics/utilities/functional/test.cu

```diff
@@ -0,0 +1,33 @@
+#include <cuda_runtime.h>
+#include "../src/serac/physics/utilities/functional/tuple_arithmetic.hpp"
+
+using namespace serac;
+
+__global__ void kernel() {
+
```

Comment:
This doesn't appear to test anything other than that the code compiles, should it be deleted?

Reply:
Good catch.